NorthstarDedicatedTest/ExploitFixes.cpp

```diff
@@ -431,7 +469,4 @@ void ExploitFixes::LoadCallback(HMODULE baseAddress)
 
 	ns_exploitfixes_log =
 		new ConVar("ns_exploitfixes_log", "1", FCVAR_GAMEDLL, "Whether to log whenever ExploitFixes.cpp blocks/corrects something");
-
-	HookEnabler hook;
-	ENABLER_CREATEHOOK(hook, (char*)baseAddress + 0x2a8a50, &GetEntByIndexHook, reinterpret_cast<LPVOID*>(&GetEntByIndex));
```

Comment:
This strips fixes by #191

Reply:
It was just moved to further up, using KHOOK:

https://github.com/R2Northstar/NorthstarLauncher/blob/561f97c223c2a71355592116bfa638690aebf6d2/NorthstarDedicatedTest/ExploitFixes.cpp#L74-L81

Reply:
Oh that's where it went. Thanks for linking <3